lib/organization.js

```diff
@@ -76,12 +84,14 @@ const insertNewOrganization = async function (claims) {
 const ensureFreshOrganizationData = async function (organization, claims) {
   const properties = [
     { predicate: 'skos:prefLabel', oldValue: organization.name, newValue: claims[AUTH_ORG_NAME_CLAIM] },
+    { predicate: 'adms:status', newValue: ACCESS_ALLOWED_STATUS_URI }
```

Comment:
Waarom is dit nodig als je de status zijn in de `INSERT DATA`? Ga je hiermee de status niet ongewild overschrijven?

Reply:
Ik ging ervan uit dat de status optioneel was, en dit diende dan om toch de ALLOWED status weg te schrijven. Deze functie zouden we niet mogen hitten indien de we al bepaald hebben dat de {user,membership,organization} geblokkeerd is.

Maar akkoord dat als status verplicht moet bestaan, dat dit dan gewoon overbodig is.